etc/picongpu/titan-ornl/cpu_picongpu.profile.example

```diff
@@ -0,0 +1,107 @@
+# Name and Path of this Script ############################### (DO NOT change!)
+export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+
+# User Information ######################################### (edit those lines)
+#   - automatically add your name and contact to output file meta data
+#   - send me a mail on job (b)egin, (e)nd, (a)bortion or (n)o mail
+export MY_MAILNOTIFY="n"
+export MY_MAIL="someone@example.com"
+export MY_NAME="$(whoami) <$MY_MAIL>"
+
+# Project Information ######################################## (edit this line)
+#   - project account for computing time
+export proj=<yourProject>
+
+# Text Editor for Tools ###################################### (edit this line)
+#   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
+#export EDITOR="nano"
+
+# basic environment ###########################################################
+source /opt/modules/3.2.6.7/init/bash
+module swap PrgEnv-pgi PrgEnv-gnu
+module swap gcc gcc/7.3.0
+
+# Compile for CLE nodes
+#   (CMake likes to unwrap the Cray wrappers)
+export CC=$(which cc)
+export CXX=$(which CC)
+export FC=$(which ftn)
+#export LD="/sw/xk6/altd/bin/ld"
+
+# symbol bug work around (should not be required)
+#MY_CRAY_LIBS=/opt/gcc/7.3.0/snos/lib64
+#export LD_PRELOAD=$MY_CRAY_LIBS/libstdc++.so.6:$LD_PRELOAD
+#export LD_PRELOAD=$MY_CRAY_LIBS/libgomp.so.1:$LD_PRELOAD
+#export LD_PRELOAD=$MY_CRAY_LIBS/libgfortran.so.3:$LD_PRELOAD
+
+# required tools and libs
+module load git
+module load cmake3/3.11.3
+# might fail to link with missing symbols:
+#   C++11 module rebuild pending [CCS #389072]
```

Comment:
note: better compile your own boost for now, see instructions here:
https://github.com/ComputationalRadiationPhysics/picongpu/blob/b2188482c7f8b325901ad8ce4718b362772bf1e9/INSTALL.rst#boost